lib/blocks/blockreader_test.go

```diff
@@ -136,7 +147,7 @@ func TestBlockDetection(t *testing.T) {
 			Log:      log,
 			ReadOnly: true,
 			LineRead: ReaderIgnore,
-			BlockRead: func(br *Reader, i int, b string) error {
+			BlockRead: func(br *Reader, i int, b string, r bool) error {
```

Comment:
lets be explicit on what the boolean does
```suggestion
			BlockRead: func(br *Reader, i int, b string, preserveIndent bool) error {
```

Reply:
:heavy_check_mark: